### PR TITLE
Removes die of fate from xenoarch

### PR DESCRIPTION
**Pull request by IrkallaEpsilon**
Nope

## About The Pull Request

Call it "No fun allowed". It happened far too often the past couple days. People throw the dice making it essentially not disappear and multi use. Since that is still an issue on TG and theres no fix I could simply try and port over we go by removing this thing from the loot table. People keep trying to roll for wizard while naked and having a syringe to remove monkeyfication on it while having a backup clone. Wizard is ultra rare for a reason. 

## Why It's Good For The Game

The fact that the die of fate is multi use when thrown instead of used in hand is bad. This removes the dreaded thing from loot tables. 

Other options I considered: Go on and comment out every 'bullshit' spell wizards possess. 

## Changelog
:cl:
del: Die of fate xenoarch loot.
/:cl:

### CODE REVIEW

**Comment by ShadeAware** [comment]
Just make it single use ffs

**Comment by MLGTASTICa** [comment]
just make it one use

**Comment by ThatGuyFromThatPlace** [comment]
I've witnessed the LRP and stupidity the dice causes. Everyone throwing it taking turns as if it doesn't have a chance to kill them, possibly dusting them, or, in the case of the previous example I saw, killing themselves and several bystanders thanks to a creature swarm around others

**Review by SarmentiCampbell — CHANGES_REQUESTED** [review]
Don't be dumb and just get the one use dice instead of straight up removing it.
/obj/item/dice/d20/fate/one_use

**Comment by SpaceVampire** [comment]
Yeah I agree with everyone else, just make it single use.

File: modular_skyrat/code/modules/research/xenoarch/artifact_list.dm [hunk 1]
```diff
@@ -49,7 +49,6 @@ GLOBAL_LIST_INIT(ult_artifact,list(	/obj/item/nuke_core/supermatter_sliver=1,
 									/obj/item/gun/medbeam=1,
 									/obj/item/storage/box/syndie_kit/imp_storage=1,
 									/obj/item/pen/edagger=1,
```
Comment (Azarak) [inline modular_skyrat/code/modules/research/xenoarch/artifact_list.dm:51]:
```suggestion
									/obj/item/pen/edagger=1,
									/obj/item/dice/d20/fate/one_use=1,
```

Comment (Azarak) [reply modular_skyrat/code/modules/research/xenoarch/artifact_list.dm:51]: One use should be fine

**Review by Azarak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**